readme.md

```diff
@@ -13,7 +13,7 @@ $ npm install --save is-binary-path
 ## Usage
 
 ```js
-var isBinaryPath = require('is-binary-path');
+const isBinaryPath = require('is-binary-path');
```

Comment:
Can you also update `http://sindresorhus.com` to https and remove the backticks from the Related links?